solana/solana-ibc/programs/solana-ibc/src/error.rs

```diff
@@ -10,9 +10,12 @@ pub(crate) enum Error {
     /// Internal error which ‘should never happen’.
     Internal(&'static str),
 
-    /// Error handling an IBC request.
+    /// Error handling an IBC deliver packet request.
     RouterError(ibc::core::RouterError),
 
+    /// Error handling an IBC send packet request
+    ContextError(ibc::core::ContextError),
+
```

Comment:
Let’s wait for https://github.com/ComposableFi/emulated-light-client/pull/128 which will cover this.

Reply:
got it.